File: DTOs/Sources/MeetingServiceDTOs/GetAttendanceDTO.swift
import Foundation

public struct GetAttendanceDTO: Codable {
    public var meetingId: UUID
    public var identity: GetIdentityDTO
    public var status: AttendanceStatus?
    
    public init(meetingId: UUID, identity: GetIdentityDTO, status: AttendanceStatus? = nil) {
        self.meetingId = meetingId
        self.identity = identity
        self.status = status
    }
}

public enum AttendanceStatus: String, Codable {
    case present
    case absent
    case accepted
}

File: android/dtos/MeetingServiceDTOs/GetAttendanceDTO.kt
package net.ipv64.kivop.dtos

data class GetAttendanceDTO (
    var meetingId : Uuid
    var identity : GetIdentityDTO
    var status : AttendanceStatus?
    var itsame : Boolean // it's-a me
)
public enum class AttendanceStatus {
    present,
    absent,
    accepted,
)
